tools/winlibs.R

```diff
@@ -1,8 +1,19 @@
-# Build against mingw-w64 build of quantlib
-if (!file.exists("../windows/quantlib-1.29/include/ql/quantlib.hpp")) {
-  download.file("https://github.com/rwinlib/quantlib/archive/v1.29.zip", "quantlib-1.29.zip", quiet = TRUE)
+if(!file.exists("../windows/quantlib/include/ql/quantlib.hpp")){
+  unlink("../windows", recursive = TRUE)
+  url <- if(grepl("aarch", R.version$platform)){
+    "https://github.com/r-windows/bundles/releases/download/quantlib-1.32/quantlib-1.32-clang-aarch64.tar.xz"
+  } else if(grepl("clang", Sys.getenv('R_COMPILED_BY'))){
+    "https://github.com/r-windows/bundles/releases/download/quantlib-1.32/quantlib-1.32-clang-x86_64.tar.xz"
+  } else if(getRversion() >= "4.3") {
+    "https://github.com/r-windows/bundles/releases/download/quantlib-1.32/quantlib-1.32-ucrt-x86_64.tar.xz"
+  } else {
+    "https://github.com/rwinlib/quantlib/archive/v1.29.tar.gz"
+  }
+  download.file(url, basename(url), quiet = TRUE)
```

Comment:
Thanks for providing 1.32.  The new version (as of this week ...) is 1.33.  RQuantLib builds without changes.  If you have spare cycles to roll to 1.33 ...

Reply:
Sorry not right now, it was already a pain to build this one.

Reply:
I complete understand.  Let's wait for the next (I think: bi-monthly) release or wait til things break.  Being a release or two or three behind is far from deadly.

Really appreciate the help with the libraries on Windows.